app/views/content/non-uk-teachers/get-an-international-relocation-payment.md

```diff
@@ -131,17 +143,25 @@ Examples of eligible languages commonly taught in English state secondary school
 * Russian
 * Spanish
 
-State schools are schools funded by the UK government. Secondary schools teach children aged 11 to 16 or 11 to 18. You will not be eligible if you teach in a school for pupils aged 16 to 18 only.
+Teaching English is not classed as an eligible subject for the IRP.
+
+You will not be eligible if you are:
 
-Ongoing or permanent contracts lasting longer than 1 year are also eligible.
+* employed by a recruitment agency – an agency can find you a job in a school, but your contract must be with the school and your salary must be paid directly to you by the school, not by the recruitment agency
 
-[Check the type and age range of the school](https://www.get-information-schools.service.gov.uk/) where you’ll be teaching. You can also find out about the [different types of state school in England](https://www.gov.uk/types-of-school).
+* teaching in Wales, Scotland or Northern Ireland
 
-### 2. Type of visa for teachers
+* a UK citizen, including citizens of Wales, Scotland and Northern Ireland
 
-You must come to England on one of the visas listed here. Most teachers will need a Skilled Worker visa.
+* an Irish citizen
 
-For the following visas, you must have applied for your visa from outside the UK:
+If you are unsure about meeting any of the eligibility requirements, email irp.mailbox@education.gov.uk for advice before planning your move.
+
+#### Type of visa
+
+You will need to come to England on a visa listed in this section to be eligible for IRP. You must have applied for your visa from outside the UK.
+
+Most teachers will need a Skilled Worker visa. Other eligible visas are:
```

Comment:
this doesn't make sense to say `Other eligible visas` and then lead with `Skilled Worker visa`?

```suggestion
Most teachers will need a Skilled Worker visa. Eligible visas are:
```